nix/pkgs/images/default.nix

```diff
@@ -58,6 +58,7 @@ let
     # Script doesn't need to be used with main branch `--alias-tag <main-branch-style-tag>`.
     # The repo chart is already prepared.
     if [[ "$(semver validate ${tag})" == "valid" ]] &&
+      [[ ! ${tag} =~ ^(v?[0-9]+\.[0-9]+\.[0-9]+-pin-(dev|devrel).([0-9]+))$ ]] &&
```

Comment:
nit: Can we keep it similar to what we follow for locals? i.e develop and prerelease, rather than dev and devrel? Just to keep things similar.

Reply:
but this is not the same though, that's just vXXX-(develop|prerelease), doesn't have the pinning;
though I agree we should change mayastor 0.0.0 to v$next-develop and the release to -prerelease to keep it consistent.
and once we change, it won't clash with the pinning, should we ever place them on the same repo